app/assets/stylesheets/_forms.scss

```diff
@@ -1,3 +1,15 @@
+.elements-error-summary {
+  border: 5px solid #b10e1e;
```

Comment:
There's a scss file for colours (which comes from using govuk_frontend_toolkit)
For example, that colour is specified here: https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/colours/_palette.scss#L21